data/world/resources/coal/init.lua

```diff
@@ -103,7 +103,7 @@
 
 push_textdomain("world")
 
-pics_dir = path.dirname(__file__) .. "pics/"
+local pics_dir = path.dirname(__file__) .. "../pics/"
```

Comment:
Since every resource now has its own folder, it would be much clearer to have the related pictures directly in there. 
Or are they referenced from somewhere else?